src/fu-util.c

```diff
@@ -3386,6 +3386,46 @@ fu_util_sync_bkc(FuUtilPrivate *priv, gchar **values, GError **error)
 	return fu_util_prompt_complete(priv->completion_flags, TRUE, error);
 }
 
+static gboolean
+fu_util_security_modify_bios_attr(FuUtilPrivate *priv, FwupdSecurityAttr *attr, GError **error)
+{
+	g_autoptr(GString) body = g_string_new(NULL);
+	g_autoptr(GString) title = g_string_new(NULL);
+
+	g_string_append_printf(title,
+			       "%s: %s",
+			       /* TRANSLATORS: title prefix for the BIOS settings dialog */
+			       _("Configuration Change Suggested"),
+			       fwupd_security_attr_get_title(attr));
+
+	g_string_append(body, fwupd_security_attr_get_description(attr));
+	g_string_append(body, "\n\n");
+	g_string_append_printf(body,
+			       /* TRANSLATORS: the % is the value, e.g. "True" or "Windows10" */
+			       _("This tool can change the BIOS setting to '%s' automatically, "
+				 "but it will only be active after restarting the computer."),
```

Comment:
how about if you tell them what it's current set to?  Not sure if it's useful so feel  free to waive.

Reply:
That's more of a designer question. I suppose if we're asking the user to potentially change it back we should tell then what it's set to. Aren't some of the BIOS attr strings localised too?

Reply:
Yeah that was my thought; they'll need to know what to set it back to if it was changed.  In terms of localization of values that would be OEM dependent if they do that.

Reply:
Gah, so I tried this, and of course got the policykit prompt -- which is really jarring and confusing.

Reply:
Hmm, in general we shouldn't be exposing what BIOS settings are for _everything_ to non-root.  But if we're already exposing that a particular current setting is what leads to a potential security vulnerability perhaps we should allow to check those particular settings?  Which I guess comes back to the question of storing some more information in the security attribute.

Reply:
Maybe `s\fwupd_security_attr_get_bios_attr_value\fwupd_security_attr_get_bios_attr_target_value` and then create `fwupd_security_attr_get_bios_attr_current_value()`? I'm nearly OUTATIME for today, but I can do it in a couple of weeks if needed.

Reply:
Yeah that's what I'm thinking, and only because of the above that it's okay.